packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -458,6 +468,21 @@ describe.concurrent(`E2E: Web frameworks`, () => {
 	});
 });
 
+const storeDiff = async (framework: string, projectPath: string) => {
+	if (!process.env.SAVE_DIFFS) {
+		return;
+	}
+
+	const outputPath = join(getDiffsPath(), `${framework}.diff`);
+
+	const output = await runCommand(["git", "diff"], {
+		silent: true,
+		cwd: projectPath,
+	});
+
+	writeFile(outputPath, output);
```

Comment:
are you missing an `await` here?

Reply:
Nope, uses `writeFileSync` under the hood.